Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN apt-get update && apt-get install -y \
 	&& apt-get clean \
 	&& rm -rf /var/lib/apt/lists/*
 
-RUN pip install --upgrade pip && pip install pytest lxml cssselect pillow==4.1.0
+RUN pip -vv install --upgrade pip==9.0.1 && pip -vv install pytest==3.6.0 lxml cssselect pillow==4.1.0
```

Comment:
Just curious, what was the problem using pip 10?

Reply:
This link helps you to understand easily. There is a big change on pip 10 they changed basic file structure of it and it causes error for apt distribution packages. Current temporary solution requires code modification or set pip 9.
https://github.com/pypa/pip/issues/5240

Reply:
Thanks, but it looks not recommended to use pip 9:
https://github.com/pypa/pip/issues/5240#issuecomment-383303439
and instead they suggested some workarounds:
https://github.com/pypa/pip/issues/5221#issuecomment-382069604

Did you try those, like `pip install --upgrade --user pip` or `hash -r pip`?

It's optional (so far we don't see any issue using pip 9), but would be nice if we can use the latest version.

Reply:
I tested different way to avoid apt package today and it's succeeded. I'll push that version which uses pip 10.

Reply:
Great, thanks!